src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionEndpoint.java

```diff
@@ -255,7 +255,7 @@ public ActionDTO updateAction(
   @RequestMapping(value = "/rerun", method = RequestMethod.PUT)
   public ResponseEntity<List<UUID>> rerunAction(
       @RequestParam(value = "actionIds") @Valid List<UUID> actionIds) throws CTPException {
```

Comment:
This should also be `value = "actionId"` singular not plural. The parameter variable is named correctly though